integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/nhttp/transport/test/ESBJAVA3689AccessMIMEMessageContentTestCase.java

```diff
@@ -86,13 +86,10 @@ public void accessMIMEMessages() throws Exception {
                 + "+CiAgICA8dG9rZW4" + "+d3NvMl9hY2Nlc3NfMDAxPC90b2tlbj4KPC9taW1lPg==</data></mediate></soapenv:Body"
                 + "></soapenv:Envelope>";
 
-        LogEvent[] logs = logViewer.getAllSystemLogs();
         boolean LogFound = false;
-        for (LogEvent log : logs) {
-            if (log.getMessage().contains(expectedMessage)) {
+            if (carbonLogReader.getLogs().contains(expectedMessage)) {
```

Comment:
It seems the indentation is messed. Shall we fix?

Reply:
have we fixed the formatting issues?

Reply:
has been fixed for a different issue